app/routes/my_updates.js

```diff
@@ -0,0 +1,22 @@
+import AuthorizeRoute from "./authorize";
```

Comment:
Can we make parent controller for this? And just extend that? Code looks same at 2 places?

CC: @Darshan-Chauhan

Reply:
sure @namrataukirde.